docs/notebooks/langmuir_samples/_generate_noisy.ipynb

```diff
@@ -0,0 +1,104 @@
+{
```

Comment:
<p>Because of course you would <code>validate_quantities</code> on a one-off noise addition :D I laugh, but it's inspiring and a good habit to incorporate.</p><p> </p><p>In that case, I want a one-liner docstring though ;)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaPy/PlasmaPy/pull/889/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
I laughed out loud at the "one-liner docstring" comment.  I totally deserve that. 🤣